model/src/main/kotlin/config/SnippetChoice.kt

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (C) 2023 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.model.config
+
+import org.ossreviewtoolkit.model.TextLocation
+
+/**
+ * A snippet choice for a given source file.
+ */
+data class SnippetChoice(
+    /**
+     * The source file criteria for which the snippet choice is made.
+     */
+    val given: Given,
+
+    /**
+     * The snippet criteria to make the snippet choice.
+     */
+    val choice: Choice
+)
+
+data class Given(
```

Comment:
Since `Given` is a pretty generic class name (and also `Choice` below is), I wonder whether we should move these to a `snippet` sub-package of `config` to avoid name clashes in the future.